packages/share-transfer/src/ShareTransferModule.ts

```diff
@@ -72,7 +76,7 @@ class ShareTransferModule implements IModule {
     const rawShareTransferStorePointer = metadata.getGeneralStoreDomain(this.moduleName) as ShareTransferStorePointerArgs;
     let shareTransferStorePointer: ShareTransferStorePointer;
     if (!rawShareTransferStorePointer) {
-      shareTransferStorePointer = { pointer: new BN(generatePrivate()) };
+      shareTransferStorePointer = { pointer: new BN(getEncryptionPrivateKey(generatePrivate(metadata.keyType), KeyType.secp256k1)) };
```

Comment:
why the pointer need to be typed?
pointer can be explicit to be a secp256k1 type.

Reply:
only the share that is being transfer need to be typed. 
other pointer, encryption can be done in secp256k1 only

Reply:
@metalurgical

Reply:
@ieow  Oh, I just though it best to be kept consistent everywhere. This would still be be correct to do, no?